aboutme.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About me!</title>
+    <link rel="stylesheet" href="Aboutme.css">
+  </head>
+  <body>
+    <h1>Day 2 assingment About me!</h1>
+    <p><strong> Hello! My name is Adan Rabadan.</strong><br />I'm atteding Code fellows to become a software developer.
```

Comment:
In the future, try to steer away from the `<br />` tag and towards putting paragraphs in their own `<p>` tags